swap_meet/vendor.py

```diff
@@ -1,2 +1,24 @@
+from .item import Item
+
 class Vendor:
-    pass
+    def __init__(self, inventory = []):
+        self.inventory = inventory
+
+    def add(self, item):
+        updated_inventory = self.inventory.append(item)
+        return item
+
+    def remove(self, item):
+        if item in self.inventory:
+            updated_inventory = self.inventory.remove(item)
+        else: 
+            item = False
+        return item
+
+    def get_by_category(self, category):
+        items = []
+        # [items.append(item) for item in self.inventory if item.category == category]
```

Comment:
This syntax for a list comprehension will function the same as your classic for loop.

```suggestion
        # items = [item in self.inventory if item.category == category]
```